modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -47,7 +51,11 @@ ifeval::["{context}" == "about-openshift-sdn"]
 
 |IPsec encryption|Not supported|Supported
 
-|IPv6|Not supported|Supported ^[3]^ ^[4]^
+|IPv4-primary dual-stack|Supported|Supported ^[3]^
+
+|IPv6 single-stack networking|Supported ^[4]^|Unsupported
+
+|IPv6 IPv6-primary dual-stack networking|Supported ^[5]^|Unsupported
```

Comment:
To be consistent with 'IPv4-primary dual-stack' above, this can be 'IPv6-primary dual-stack'

Reply:
:heavy_check_mark: